src/promptflow/promptflow/_core/_errors.py

```diff
@@ -139,6 +139,17 @@ class MetaFileReadError(GenerateMetaUserError):
     pass
 
 
+class GenerateMetaSystemError(SystemErrorException):
+    """Base system exception raised when failed to validate tool."""
+
+    def __init__(self, **kwargs):
+        super().__init__(target=ErrorTarget.RUNTIME, **kwargs)
```

Comment:
Not sure when do we need to assign error target as `Runtime` in github.

Reply:
And maybe move to `src/promptflow/promptflow/executor/_service/_errors.py`?

Reply:
Change to executor

Reply:
I think this error is more related to gen meta, maybe it will be used in tool_meta_generator in the future.